server/src/main/java/org/opensearch/telemetry/tracing/AttributeNames.java

```diff
@@ -40,6 +40,11 @@ private AttributeNames() {
      */
     public static final String HTTP_URI = "http.uri";
 
+    /**
+     * Http Request Query Path.
+     */
+    public static final String HTTP_REQ_QUERY_PARAMS = "http.query_params";
```

Comment:
We would need to use Semantic Conventions for HTTP Spans (https://opentelemetry.io/docs/specs/semconv/http/http-spans/):

```suggestion
    public static final String HTTP_REQ_QUERY_PARAMS = "url.query";
```